ocs_ci/deployment/vmware.py

```diff
@@ -258,6 +268,26 @@ def get_compute_vms(self, dc, cluster):
             )
             return [vm for vm in vms if "compute" in vm.name or "rhel" in vm.name]
 
+    def get_sno_vms(self, dc, cluster):
```

Comment:
I suggest not to have a specific function that returns VMs with `sno` string in their name, but rather add one more argument to get_compute_vms() which will allow querying for a that string in the VM name

Reply:
We have already a function that call compute vm but it is looking for a string in vcenter for "compute". My nodes are called "sno-edge-0".
What I'll do is change my function to get a string to search for.